recipe/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "st-theme" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/st-theme-{{ version }}.tar.gz
+  sha256: ca97aece1a48ded6e83fd742c27cb0851e1bce2100ab4b6c37c7b6e003b65b42
+
+build:
+  entry_points:
+    - st-theme = streamlit_theme:print_version
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [s390x or py<=38]
```

`setup.py` says `python_requires=">=3.8"`?:

```suggestion
  skip: true  # [s390x or py<38]
```